### DISCUSSION TITLE
During the search of a char the focus should be moved to the first element

### DISCUSSION THREAD

**Discussion by MarcAnt01** [character-map-uwp/Character-Map-UWP#192] [Ideas]
This way I could just press enter from keyboard, without having to move the cursor

**Comment by JohnnyWestlake** [comment]
The immediate issue with this is it prevents typing more characters.

However you can currently highlight *any* and all results in the search result list by using the up & down arrow keys on your keyboard, if this satisfies your needs.

**Comment by MarcAnt01** [comment]
> The immediate issue with this is it prevents typing more characters.
> 
> However you can currently highlight _any_ and all results in the search result list by using the up & down arrow keys on your keyboard, if this satisfies your needs.

Sorry, but I didn't quite explain what I meant

**Comment by MarcAnt01** [comment]
The thing is that after pressing enter I expect it to perform a kind of auto-completion as if I click on the first result, does that sound reasonable to you?

**Comment by JohnnyWestlake** [comment]
From precursory investigating I can't find any simple way of programmatically moving the focus down without causing the popup result list to close. So I'll leave this on the investigation list for now
